func-consumption/Dockerfile

```diff
@@ -1,12 +1,5 @@
-### DO NOT RUN THIS FILE DIRECTLY
-### execute ../run-docker-compose-travis.sh in order to ensure tslib dependencies are copied in prior to build
-FROM baldy/mtc-azure-functions-base:latest
-
-SHELL ["/bin/bash", "-c"]
-
-RUN mkdir -p /func-app
-WORKDIR /func-app
-COPY . .
-
+FROM mtc_functions_base:latest
```

Comment:
where is it sourcing `mtc_functions_base` from? the owner prefix has been dropped...

Reply:
See the docker-compose.yml file for details - it gets built on the fly.